README.md

```diff
@@ -77,7 +81,8 @@ Use one or more of following keys to fire your hotkeys.
 - ctrl
 - alt
 - shift
-- meta (windows / command)
+- command (MacOS)
```

Comment:
Why to divide meta into windows and command?

Reply:
I'm so sorry that I haven't had enough time to maintain my repo recently, I found it's the same issue to above one. Ignore it.